tsconfig.full.json

```diff
@@ -8,7 +8,8 @@
     "noUnusedParameters": false,
     "noImplicitAny": false,
     "noImplicitThis": false,
-    "noEmitOnError": false
+    "noEmitOnError": false,
+    "lib": ["dom", "es2015"]
```

Comment:
why is `dom` needed?

Reply:
`@types/protobufjs` requires it. Sorry, was about to mention that and then forgot.

Reply:
Why is `@types/protobufjs` needed?

Reply:
It's a transitive dependency for `grpc` types.

Reply:
*sigh*.

Reply:
I have two questions:
* Why is grpc leaking this to its users? `grpc-node` does not support the browser and requiring their users to have browser specific TypeScript config seems "wrong".
* I am fairly convinced that this module wouldn't leak the dependency on `dom` to our users, but can you confirm?

Reply:
> Why is grpc leaking this to its users? `grpc-node` does not support the browser and requiring their users to have browser specific TypeScript config seems "wrong".
That we currently expose `protobufjs` options to the user is a symptom of the problem that is exactly what `@grpc/proto-loader` is trying to solve. At best we can re-define the options in the `grpc` types. (Actually, if you think that's a good idea I can go ahead and do that.)
> I am fairly convinced that this module wouldn't leak the dependency on `dom` to our users, but can you confirm?
That's a valid concern -- it doesn't because the package currently doesn't publish types for the gRPC patching plugin.